src/Microsoft.CodeAnalysis.Analyzers/ReleaseTrackingAnalyzers.Help.md

```diff
@@ -0,0 +1,100 @@
+# Release tracking analyzer
+
+Release tracking analyzer enables third party analyzer packages to define analyzer releases with associated versions. Each release can track the following changes:
+1. Additions: Set of new analyzer rules that shipped for the first time in this release.
+2. Removals: Set of old analyzer rules that shipped in an earlier release, but are removed starting this release.
+3. Changes: Set of existing analyzer rules that shipped in an earlier release, but one of the following attributes of the diagnostic changed in this release:
+   1. Category of the diagnostic.
+   2. Default severity of the diagnostic
+   3. Enabled by default status.
+
+This analyzer expects that the thirdparty analyzer project has two additional files, namely `AnalyzerReleases.Unshipped.md` and `AnalyzerReleases.Shipped.md`, to track this release specific metadata.
```

Comment:
>the [](start = 27, length = 3)

Should mention that each analyzer project needs its own pair of files. If there multiple analyzer assemblies in a package, then they'll each need their own unique pair, right?

Reply:
Yes, that is correct.